src/test/unit/process-profile.test.js

```diff
@@ -229,3 +231,56 @@ describe('gecko counters processing', function() {
     expect(processedCounters[1].sampleGroups.samples.time).toEqual(offsetTime);
   });
 });
+
+describe('serializeProfile', function() {
+  it('should produce the same profile in a roundtrip', async function() {
+    const profile = processProfile(createGeckoProfile());
+    const serialized = serializeProfile(profile);
+    const roundtrip = await unserializeProfileOfArbitraryFormat(serialized);
+    // FIXME: Uncomment this line after resolving `undefined` serialization issue
+    // See: https://github.com/devtools-html/perf.html/issues/1599
+    // expect(profile).toEqual(roundtrip);
+
+    const secondSerialized = serializeProfile(roundtrip);
+    const secondRountrip = await unserializeProfileOfArbitraryFormat(
+      secondSerialized
+    );
+    expect(roundtrip).toEqual(secondRountrip);
+  });
```

Comment:
For the code style on tests, there is a space between `describe` and `it` blocks.